guides/common/modules/proc_synchronizing-content-over-a-network.adoc

```diff
@@ -0,0 +1,85 @@
+[id="synchronizing-content-over-a-network_{context}"]
+= Synchronizing Content Over a Network
+
+Configure a downstream {ProjectServer} to synchronize repositories from a connected upstream {ProjectServer} over HTTPS.
+
+.Prerequisites
+* A network connection exists between the upstream {ProjectServer} and the downstream {ProjectServer}.
+* You have imported the subscriptions manifest on both the upstream and downstream {ProjectServer}.
```

Comment:
```suggestion
* You have imported the subscription manifest on both the upstream and downstream {ProjectServer}.
```

Reply:
Damn, I have duplicate procedures by mistake. This one shouldn't be here anymore. Sorry about that. But I'll fix this in the proper procedure, thanks.

Reply:
Not to my knowledge @Lennonka 
For this reason, we are quite strict about deleting things we aren't using anymore as part of a PR. 
People try to raise PRs where they comment things out and our general requirement is to delete. 
I don't have a technical solution for unused modules.

Reply:
We do now: https://github.com/theforeman/foreman-documentation/pull/1296